dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -528,19 +707,29 @@ private void launch(final List<String> args) {
         if (args.isEmpty() || containsHelpRequest(args)) {
             launchHelp();
         } else {
-            final String descriptor = optVal(args, "--descriptor", CWL_STRING);
-
-            if (descriptor.equals(CWL_STRING)) {
-                try {
-                    launchCwl(args);
-                } catch (ApiException e) {
-                    exceptionMessage(e, "api error launching workflow", Client.API_ERROR);
-                } catch (IOException e) {
-                    exceptionMessage(e, "io error launching workflow", IO_ERROR);
+            if (args.contains("--local-entry")) {
+                final String descriptor = optVal(args, "--descriptor", null);
+                List<String> argsList = new ArrayList<>();
```

Comment:
Ah ok, I would add a new method signature along the lines of `private void launchCwl(final String entry, final List<String> args)` and re-use most of the existing method (i.e. not copy-and-paste) instead of creating a new list this way.